### PR TITLE
feat:build address management feature

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link : https://deploy-preview-7--stunnershub.netlify.app/pages/cart/cart.html

### CODE REVIEW

File: pages/cart/cart.css [hunk 1]
```diff
@@ -40,6 +40,7 @@ body {
   justify-content: center;
   background: var(--white);
   box-shadow: 0px 2px 3px 0 rgb(0 0 0 / 10%);
```
Comment (ckahub) [inline pages/cart/cart.css]: use rem everywhere

File: pages/cart/cart.css [hunk 2]
```diff
@@ -161,14 +163,117 @@ body {
...
+.address-sec {
+  width: 70%;
+}
+
+.modal{
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  overflow: hidden;
+  display: none;
```
Comment (ckahub) [inline pages/cart/cart.css:203]: 2 display property in one class

File: pages/cart/cart.css [hunk 3]
```diff
@@ -161,14 +163,117 @@ body {
...
+  font-size: 1.1rem;
+  color: var(--bg-color);
+  font-weight: 500;
+  margin-bottom: 1rem;
+}
+
+.cancel-btn{
+  background-color: var(--product-bgcolor);
+  color: var(--bg-color);
+  border-radius: 5xp;
```
Comment (ckahub) [inline pages/cart/cart.css]: use rem for border radius

File: pages/cart/cart.css (hunk 1 above)
Comment (ckahub) [inline pages/cart/cart.css]: use hex for color

**Review by ckahub — COMMENTED** [review]
do the needful

File: pages/cart/cart.css (hunk 3 above)
Comment (ishitakeshawani) [reply pages/cart/cart.css]: I do not t use rems for a value less than 10px.

File: pages/cart/cart.css (hunk 2 above)
Comment (ishitakeshawani) [reply pages/cart/cart.css:203]: done

File: pages/cart/cart.css (hunk 1 above)
Comment (ishitakeshawani) [reply pages/cart/cart.css]: done

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

**Pull request merged**